fragments/subs.js

```diff
@@ -19,6 +19,13 @@ export const SUB_FIELDS = gql`
     moderated
     moderatedCount
     meMuteSub
+    user {
+      name
+      id
+      optional {
+        streak
+      }
+    }
```

Comment:
Use the fragment below this one instead if you need the user. This fragment is intended to be used without the additional db dip.